packages/dd-trace/src/format.js

```diff
@@ -123,27 +108,31 @@ function extractAnalytics (trace, span) {
   }
 }
 
-function addTag (meta, key, value, seen) {
+function addTag (meta, metrics, key, value, seen) {
   switch (typeof value) {
     case 'string':
       meta[key] = value
       break
+    case 'number':
+      if (isNaN(value)) break
```

Comment:
we also need to check that `absolute_value(value) < 2**53`, but only for integers, floats are fine.

cc @lraucy

Reply:
Integers cannot go beyond that in JavaScript, so no check is needed.

Reply:
I'm also pretty sure this limitation actually comes from something JavaScript.

Reply:
oh nice, JavaScript to the rescue!

Reply:
What about the bigint type? Can we have bigints `< 2**53` in javascript? If that's a possible use case, it would make sense to put them into `metrics` as numbers. And keep the bigints in `meta` when they're above or equal to `2**53`.

I hope what I'm saying here makes sense, I don't know javascript very much.

Reply:
Let's keep them as strings to avoid weird issues where you sometimes have a BigInt too large and sometimes not in the same tag. In general BigInts will only be used when the value will be 64-bit or higher anyway.